### PR TITLE
Support tags column in View/Print Issues, Csv/Excel Export

### PR DESCRIPTION
**Pull request by lukelee1124**
Fixes [#9096](https://www.mantisbt.org/bugs/view.php?id=9096)

### CODE REVIEW

**Comment by vboctor** [comment]
:+1:  Looks good.

File: core/excel_api.php [hunk 1]
```diff
@@ -364,6 +364,15 @@ function excel_format_fixed_in_version( BugData $p_bug ) {
 }
 
 /**
+ * Gets the formatted tags.
+ * @param BugData $p_bug A bug object.
+ * @return string the tags.
+ */
+function excel_format_tags( BugData $p_bug ) {
+	return excel_prepare_string( access_has_bug_level( config_get( 'tag_view_threshold' ), $p_bug->id ) ? tag_bug_get_all( $p_bug->id ) : '' );
```
Comment (dregad) [inline core/excel_api.php]: This line is too long, can you please split it ?

File: core/columns_api.php [hunk 2]
```diff
@@ -161,6 +161,8 @@ function columns_get_standard( $p_enabled_columns_only = true ) {
 
 	$t_columns['notes'] = null;
```
Comment (dregad) [inline core/columns_api.php]: Remove extra blank line.

Also (somewhat unrelated to this change), unless the sequence in the array is significant (didn't check), maybe consider moving these 2 assignments to line 126.

**Comment by vboctor** [comment]
Merged via 272624e117d1859bc1a75fb6eea6880fbd246695 - thanks.

**Comment by leekisang** [comment]
hi sir.
A bug occurs when "EXCEL" is generated.

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html>
<head>
	<link rel="stylesheet" type="text/css" href="http://58.225.115.230:8081/mz/css/default.css" />
	<script type="text/javascript" language="JavaScript"></script>
	<meta http-equiv="Content-type" content="text/html; charset=utf-8" />
	<meta http-equiv="Pragma" content="no-cache" />
	<meta http-equiv="Cache-Control" content="no-cache" />
	<meta http-equiv="Pragma-directive" content="no-cache" />
	<meta http-equiv="Cache-Directive" content="no-cache" />
	<meta http-equiv="Expires" content="Thu, 27 Apr 2017 00:12:12 GMT" />
	<link rel="shortcut icon" href="/mz/images/favicon.ico" type="image/x-icon" />
	<link rel="search" type="application/opensearchdescription+xml" title="MantisBT: Text Search" href="http://58.225.115.230:8081/mz/browser_search_plugin.php?type=text" />	<link rel="search" type="application/opensearchdescription+xml" title="MantisBT: Issue Id" href="http://58.225.115.230:8081/mz/browser_search_plugin.php?type=id" />	<title>MantisBT</title>
<script type="text/javascript" src="/mz/javascript/min/common.js"></script>
<script language="javascript">var loading_lang = "Loading...";</script><script type="text/javascript" src="/mz/javascript/min/ajax.js"></script>
</head>
<body>
<div align="left"><a href="my_view_page.php">Mantis Bug Tracker</a></div><table class="hide"><tr><td class="login-info-left">Logged in as: <span class="italic">shsnshsn</span> <span class="small">(이기상 - administrator)</span></td><td class="login-info-middle"><span class="italic">2017-04-27 09:12 KST</span></td><td class="login-info-right"><form method="post" name="form_set_project" action="/mz/set_project.php">Project: <select name="project_id" class="small" onchange="document.forms.form_set_project.submit();"><option value="0">All Projects</option>
<option value="27">Wearable</option>
</select> <input type="submit" class="button-small" value="Switch" /></form><a href="http://58.225.115.230:8081/mz/issues_rss.php?username=shsnshsn&amp;key=93ec8cf0bf8a3865dae8fef746102abe&amp;project_id=13">RSS</a></td></tr></table><table class="width100" cellspacing="0"><tr><td class="menu"><a href="/mz/main_page.php">Main</a> | <a href="/mz/my_view_page.php">My View</a> | <a href="/mz/view_all_bug_page.php">View Issues</a> | <a href="/mz/bug_report_page.php">Report Issue</a> | <a href="/mz/changelog_page.php">Change Log</a> | <a href="/mz/roadmap_page.php">Roadmap</a> | <a href="/mz/summary_page.php">Summary</a> | <a href="/mz/manage_overview_page.php">Manage</a> | <a href="/mz/account_page.php">My Account</a> | <a href="/mz/logout_page.php">Logout</a></td><td class="menu right nowrap"><form method="post" action="/mz/jump_to_bug.php"><input type="text" name="bug_id" size="10" class="small" value="Issue #" onfocus="if (this.value == 'Issue #') this.value = ''" onblur="if (this.value == '') this.value = 'Issue #'" />&nbsp;<input type="submit" class="button-small" value="Jump" />&nbsp;</form></td></tr></table><br /><div align="center"><table class="width50" cellspacing="1"><tr><td class="form-title">APPLICATION ERROR #1100</td></tr><tr><td><p class="center" style="color:red">Issue 0 not found.</p></td></tr><tr><td><p class="center">Please use the "Back" button in your web browser to return to the previous page. There you can correct whatever problems were identified in this error or select another action. You can also click an option from the menu bar to go directly to a new section.</p></td></tr></table></div>	<br />
	<hr size="1" />
<table border="0" width="100%" cellspacing="0" cellpadding="0"><tr valign="top"><td>	<address>Copyright &copy; 2000 - 2010 MantisBT Group</address>
	<address><a href="mailto:mantis2@mediazen.co.kr">mantis2@mediazen.co.kr</a></address>
</td><td>
	<div align="right"><a href="http://www.mantisbt.org" title="Free Web Based Bug Tracker">Powered by Mantis Bugtracker</a></div>
</td></tr></table>
</body>
</html>

**Comment by atrol** [comment]
> Copyright © 2000 - 2010 MantisBT Group
 
@leekisang you are using a complete outdated version of MantisBT.
You will hardly get any support for it.
I recommend to upgrade your installation.

**Comment by dregad** [comment]
@leekisang in addition to atrol's comment above, please note that this is not the right place to get assistance. Please see http://mantisbt.org/support.php for valid support channels.

**Pull request closed without merging**